vignettes/DataAnalysis.Rmd

```diff
@@ -189,7 +190,7 @@ chart <- Visualize_Scatter(dfFlagged, dfBounds)
 chart
 ```
 
-A full explanation of all Data Visualizations in the `{gsm}` package is outlined in the [Reporting Workflow vignette](DataReporting.html).
+A full explanation of all Data Visualizations in the `{gsm}` package is outlined in the [Reporting Workflow vignette](https://gilead-biostats.github.io/gsm/articles/DataReporting.html).
```

Comment:
```suggestion
A full explanation of all Data Visualizations in the `{gsm}` package is outlined in the `vignette("DataReporting")`.
```

This *should* automatically generate a link to the vignette in the pkgdown, and also give the user info about how to find it locally.